usdQt/hierarchyModel.py

```diff
@@ -151,7 +151,7 @@ def __OnObjectsChanged(self, notice, sender):
                     indexPath = indexPrim.GetPath()
 
                     for resyncedPath in resyncedPaths:
-                        commonPath = resyncedPath.GetCommonPrefix(path)
+                        commonPath = resyncedPath.GetCommonPrefix(resyncedPath)
```

Comment:
Is it meaningful for a path to get a common prefix with itself?  what's the result?

Reply:
Hey Matt, if you can get this last bit addressed we can get this merged in.

Reply:
I'll look at this today.  I think it's fixed locally.